redpen-cli/sample/js/test/validator-js-test.js

```diff
@@ -0,0 +1,52 @@
+/*
+tests validator.js implementation.
+
+steps to run this test:
+ 1. install mocha
+ $ npm install -g mocha
+
+ 2. run RedPen in server mode
+ $ cd $REDPEN_HOME/bin
+ $ ./redpen-server
+
+ 3. rename validator.js.example to enable the validator implementation
+ $ cd $REDPEN_HOME/sample
```

Comment:
I think there is no **sample** directory in $REDPEN_HOME. **js** directory is correct.